src/components/v5/shared/MeatBallMenu/partials/MeatBallMenuItems.tsx

```diff
@@ -0,0 +1,63 @@
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import HoverWidthWrapper from '~v5/shared/HoverWidthWrapper/HoverWidthWrapper.tsx';
+
+import { type MeatBallMenuItemsProps } from '../types.ts';
+
+export const MeatBallMenuItems: FC<MeatBallMenuItemsProps> = ({
+  items,
+  renderItemWrapper,
+  onClose,
+}) => {
+  return (
+    <ul>
+      {items.map(
+        ({
+          key,
+          label,
+          onClick,
+          icon: Icon,
+          renderItemWrapper: itemRenderItemWrapper,
+          className: itemClassName,
+        }) => (
+          <li key={key} className={clsx(itemClassName, 'flex-shrink-0')}>
+            <HoverWidthWrapper hoverClassName="w-full md:font-medium">
+              {(itemRenderItemWrapper || renderItemWrapper)(
```

Comment:
How I wish these were composed in instead :melting_face: But not in scope of this issue, would probably be way too much of a rework :D